content/beam/update/2016/10/11/strata-hadoop-world-and-beam.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
```

Comment:
FYI -- in the future, please drop the content directory from the PR. (Merging autogenerated content gets gross, so the person who does the merge will do the final generate. Instructions in contribution guide are getting updated in pull/42.)